src/daft-plan/src/partitioning.rs

```diff
@@ -150,6 +153,146 @@ impl ClusteringSpec {
     }
 }
 
+pub fn translate_clustering_spec(
```

Comment:
You're just moving the function to a new file right? No changes?

Reply:
yep, no changes